server/src/main/java/fr/uga/l3miage/tp4/repositories/CandidateRepository.java

```diff
@@ -0,0 +1,15 @@
+package fr.uga.l3miage.tp4.repositories;
+
+import fr.uga.l3miage.tp4.models.CandidateEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Repository
+public interface CandidateRepository extends JpaRepository<CandidateEntity, Long> {
+    Set<CandidateEntity> findAllByBirthDateBeforeAndHasExtraTimeFalse(LocalDate birthDate);//quatrième requête
+    Set<CandidateEntity> findAllByTestcenterCodeGre();//deuxième requête
```

Comment:
non tu ne peux pas lui donner directement une donnée mais plutot : 
```suggestion
    Set<CandidateEntity> findAllByTestcenterCode(TestCenterCode code);//deuxième requête
```